swan-lake/learn/security/writing-secure-ballerina-code.md

```diff
@@ -47,14 +47,13 @@ public remote function query(@untainted string|sql:ParameterizedQuery sqlQuery,
 
 The following example constructs an SQL query with a tainted argument:
 
+Note: Taint checker is disabled by default. This can be enabled by adding --taint-check flag to build or run command
```

Comment:
```suggestion
Note: Taint checker is disabled by default. This can be enabled by adding the `--taint-check` flag to the `build` or `run` command
```